shared/src/address/payload.rs

```diff
@@ -19,30 +73,36 @@ pub enum Payload {
     Actor([u8; PAYLOAD_HASH_LEN]),
     /// BLS key address, full 48 byte public key
     BLS([u8; BLS_PUB_LEN]),
+    /// Namespaced
+    Namespaced {
```

Comment:
Definitely hate this name. Just call it `f4`? Or...?

Also, `::Actor` is now misnamed because these are both "actor".

Maybe rename to:

- `Stable`
- `Managed` (?) idk

Reply:
`Managed` sounds more appropriate, f4 addresses aren't guaranteed to be stable.

Reply:
`Delegated`?

Reply:
`Delegate` I was hoping to use for the signature type name in account abstraction, which may cause confusion. Though account abstraction isnt finished and we can bikeshed that name as well

Reply:
Delegated matches IP address subrange delegation.